internal/api/health_check.go

```diff
@@ -20,15 +20,18 @@ type healthCheckResponse struct {
 	Description string                       `json:"description"`
 }
 
+// initHealthCheck health check endpoints on the given router.
 func initHealthCheck(apiRouter *mux.Router, context *Context) {
 	addContext := func(handler contextHandlerFunc) *contextHandler {
 		return newContextHandler(context, handler)
 	}
 
 	pluginsRouter := apiRouter.PathPrefix("/health").Subrouter()
-	pluginsRouter.Handle("", addContext(handleHealthCheck)).Methods("GET")
+	pluginsRouter.Handle("", addContext(handleHealthCheck)).Methods(http.MethodGet)
 }
 
+// handleHealthCheck responds to GET /api/v1/health,
```

Comment:
nit: was the line wrap here strictly necessary?

Reply:
Without the wrapping the line has 130 characters, which is too much IMO if it can be avoided.	But I'm 0/5, because we don't enforce line length.